src/Compilers/VisualBasic/Test/Semantic/IOperation/IOperationTests_IForEachLoopStatement.vb

```diff
@@ -0,0 +1,749 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports Microsoft.CodeAnalysis.Semantics
+Imports Microsoft.CodeAnalysis.Test.Utilities
+Imports Microsoft.CodeAnalysis.VisualBasic.Syntax
+Imports Roslyn.Test.Utilities
+
+Namespace Microsoft.CodeAnalysis.VisualBasic.UnitTests.Semantics
+
+    Partial Public Class IOperationTests
```

Comment:
>IOperationTests [](start = 25, length = 15)

Comments for C# tests are applicable to VB tests as well.
Also, control variable of a For Each loop in VB doesn't have to be a local. It could be a field, or an array element, for example. Please add tests like this. 
Also, in VB, a local, used as control variable, doesn't have to be declared by the loop. Should the API reflect the difference? Please add tests.